src/tribler-core/tribler_core/trustchain_keys.py

```diff
@@ -0,0 +1,30 @@
+import tribler_core.utilities.permid as permid_module
+from tribler_core.utilities.path_util import Path
+
+
+class TrustChainKeys:
```

Comment:
In the future, it is better to leave only the single keypair property in the code, loading either the real key or the testnet key based on the config option. AFAIK, there are no situations where *both* keys are used in the code simultaneously.